msg/en_US.utf8/utils.msg

```diff
@@ -1309,3 +1310,25 @@ valid options:\n\
                                   for flashback; default: outputs only SQL statements for flashback\n\
     --oldest                      flashback is performed in chronological order for SQL statements executed from start time to end time;\n\
                                   default: latest\n
+
+$set 59 MSGCAT_UTIL_SET_MEMMON
+1 '-a(--all)' option cannot be used with other options.\n
+2 Please set the system parameter 'memory_monitoring' in 'cubrid.conf' and restart the database server.\n
+3 Invalid argument '%1$s' is set to '-m(--module)' option.\n
+4 '-c(--tran-count)' option must be used with '-t(--transaction)' option.\n
+5 '-c(--tran-count)' option must have positive number.\n
+59 memmon cannot run as standalone mode.\n
+60 \
+memmon: Provides memory usage information of a database server process.\n\
+usage: %1$s memmon [OPTION] database-name\n\
+\n\
+valid options:\n\
+    -m, --module=TYPE             TYPE of argument\n\
+                                    0 (or all)      display detailed memory usage information of the entire registgered modules\n\
+                                    1 (or heap)     display detailed memory usage information of the heap module.\n\
+    -t, --transaction             display memory usage of active transactions with descending order of memory usage.\n\
```

Comment:
```suggestion
    -t, --transaction             display memory usage of active transactions in descending order of memory usage.\n\
```
Please use a grammar checker like Grammarly to fix typos and errors for a manual or messages for users. It has some more to fix.

Reply:
Chat GPT says using 'with' is no grammatical error but recommends using 'in'. I change it.